src/Console/GenerateCommand.php

```diff
@@ -368,9 +367,11 @@ public static function enumValues($table, $name)
         if ($table === null) {
             return "[]";
         }
-
-        $type = DB::select(DB::raw('SHOW COLUMNS FROM ' . $table . ' WHERE Field = "' . $name . '"'))[0]->Type;
-
+        if (env('DB_HOST') == 'postgres') {
```

Comment:
I don't think that's a good idea to check the database host to determine whether it's PostgreSQL.
This check should be for pgsql driver:
```if (env('DB_CONNECTION') == 'pgsql') {```